### ISSUE TITLE
Typechecks for react-rrule-builder-ts are failing during yarn build

### ISSUE THREAD

**Issue by elitsab** [dcantatore/react-rrule-builder-ts#5]
Tried to push a fix to remove the 'clearable' attributes as they are not type defined but did not have permission to create a branch.

https://github.com/search?q=repo%3Adcantatore%2Freact-rrule-builder-ts%20clearable&type=code

Could you please remove them? Thank you!

**Comment by dcantatore** [comment]
Hi @elitsab -

I'm not sure what you mean.  Are you saying - you would like start and end date clear functionality to be configurable? Other wise those slotProps are the correct types as they are.

**Comment by elitsab** [comment]
I meant to say that 'clearable' property is not known/defined. Installing the package via yarn and doing a yarn build results in errors: 
![image](https://github.com/user-attachments/assets/366a78ff-aaab-4142-ae8a-68a9816a84d1)

**Comment by elitsab** [comment]
I got access denied when I attempted to push a branch earlier. 
![image](https://github.com/user-attachments/assets/673724b2-4668-4624-b57d-0d152dd4f4b9)

**Comment by dcantatore** [comment]
I'm making a guide for contributing that should help, will be out soon. 


For the error - have you installed the peer dependencies in your project? This package relies on you having them installed. 

If so and it's still an issue can you post an example repo to reproduce the issue? Otherwise I haven't been able to reproduce it.

**Comment by elitsab** [comment]
Hi Dario! 

I apologize I did not get the chance to respond earlier! The issue turned out to be with out setup on our side as you implied! We seem to have enforced tighter TypeScript rules on our end and for some reason the skipLibCheck compiler option for TypeScript did not work so it was emitting errors from the package. I tried to isolate the issue in a clean vite + React + Typescript app and am happy to share that I was unable to reproduce the issue.

We do appreciate the documentation on how to contribute back and the datetime adapters! We also added some additional logic to convert the rrule strings to cron for our background job Hangfire runners using [this](https://gist.github.com/emanuelet/65bd1c590265b622dcc38e9b856a410f). Thank you for repo!

Any idea when you will mark your work as prod ready?
